docs/installation-guide-aws.md

```diff
@@ -0,0 +1,147 @@
+# Installation steps
+
+## Prerequisites
+
+Common:
+- `aws` cli
+
+### Create S3 bucket and DynamoDB table for terraform state
+
+```
+export CLUSTER="cluster1"
+export AWS_DEFAULT_REGION="eu-central-1"
+```
+
+```
+aws s3 mb s3://$CLUSTER-state --region $AWS_DEFAULT_REGION
+
+aws s3api put-bucket-versioning --bucket $CLUSTER-state \
+    --versioning-configuration Status=Enabled
+
+aws dynamodb create-table --region $AWS_DEFAULT_REGION \
+    --table-name $CLUSTER-lock \
+    --attribute-definitions AttributeName=LockID,AttributeType=S \
+    --key-schema AttributeName=LockID,KeyType=HASH \
+    --provisioned-throughput ReadCapacityUnits=5,WriteCapacityUnits=5
+```
+
+### Prepare terraform build directory
+
+```
+mkdir -p build
+cp -r examples/aws/example-build/* build
+cd build
+```
+
+Put proper values in `backend.tf` and make sure backend configuration linked properly.
+
+```
+cat ../platforms/azure/giantnetes/backend.tf
+```
+
+Edit `envs.sh`.
+
+```
+source envs.sh
+```
+
+NOTE: Reexecute `source envs.sh` in every new console.
+
+## Install
+
+Install consists two stages:
+- Vault (only needed because we bootstrapping Vault manually)
+- Kubernetes
+
+Master and workers will be created with in the Vault stage and expectedly will fail (and recreated later). This is done to keep single Terraform state and simplify cluster management after installation. Master and workers will be reprovisioned with right configuration in the second state called Kubernetes.
```

Comment:
with in -> within
Why do we need to create master/workers within vault stage?

Reply:
because we have one terraform manifest for whole cluster and Vault still provisioned with Ansible.